From 3a503f51a189bd5b3eb7436981e3a26816139433 Mon Sep 17 00:00:00 2001
From: Dan Williams <dan.j.williams@intel.com>
Date: Wed, 5 Oct 2016 15:54:46 -0700
Subject: libnvdimm, namespace: lift single pmem limit in scan_labels()
Git-commit: 991d9020f3e0447ea00c7c7f11fed364d977320a
Patch-mainline: v4.9-rc1
References: FATE#321135, FATE#321217, FATE#321256, FATE#321391, FATE#321393

Now that the rest of the infrastructure has been converted to handle
multi-pmem configurations, lift the artificial barrier at scan time.

Signed-off-by: Dan Williams <dan.j.williams@intel.com>
Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>

---
 drivers/nvdimm/namespace_devs.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/nvdimm/namespace_devs.c b/drivers/nvdimm/namespace_devs.c
index 54babc3..fa51d75 100644
--- a/drivers/nvdimm/namespace_devs.c
+++ b/drivers/nvdimm/namespace_devs.c
@@ -2070,9 +2070,6 @@ static struct device **scan_labels(struct nd_region *nd_region)
 				}
 			} else
 				devs[count++] = dev;
-
-			/* we only expect one valid pmem label set per region */
-			break;
 		}
 	}
 
-- 
1.8.5.6

